src/test/java/uk/ac/ebi/ega/egafuse/runner/CommandLineOptionParserTest.java

```diff
@@ -107,4 +109,22 @@ public void parser_WhenGivenMountPathNotExists_ThenThrowsException() throws IOEx
         OptionSet set = CommandLineOptionParser.buildParser().parse(args) ;        
         CommandLineOptionParser.parser(set);
     }
+    
+    @Test
+    public void parser_WhenGivenTreeOptionDisable_ThenReturnsFalse() throws IOException{        
+        final File mountFolder = temporaryFolder.newFolder("tmp", "mount");
+        String[] args = { "-m", mountFolder.toPath().toAbsolutePath().toString(), "-u", "amohan", "-p", "testpass", "-t", "disable"};
+        OptionSet set = CommandLineOptionParser.buildParser().parse(args) ;        
+        CliConfigurationValues cliConfigurationValues = CommandLineOptionParser.parser(set);
+        assertFalse(cliConfigurationValues.isTreeStructureEnable());
+    }
+    
+    @Test
+    public void parser_WhenGivenTreeOptionEnable_ThenReturnsTrue() throws IOException{        
+        final File mountFolder = temporaryFolder.newFolder("tmp", "mount");
+        String[] args = { "-m", mountFolder.toPath().toAbsolutePath().toString(), "-u", "amohan", "-p", "testpass", "-t", "enable"};
+        OptionSet set = CommandLineOptionParser.buildParser().parse(args) ;        
+        CliConfigurationValues cliConfigurationValues = CommandLineOptionParser.parser(set);
+        assertTrue(cliConfigurationValues.isTreeStructureEnable());
```

Comment:
Do we test what happens if they give us a wrong input in the -t parameter?

Reply:
Added a new test case now for this condition, i.e, invalid input in -t, it will discard that and defaults it to enable.